docs/conf.py

```diff
@@ -29,6 +29,7 @@
     'sphinx.ext.viewcode',
     'sphinxcontrib.httpdomain',
     'sphinx_rtd_theme',
+    'sphinx_automodapi.automodapi',
```

Comment:
I'm using 
```python
extensions = [
    "sphinx_rtd_theme",
    "sphinx.ext.autodoc",
    "sphinx.ext.autosummary",
]
```
Not `sphinx_automodapi`

Reply:
Oh thanks! I picked this up from PlasmaPy's issues and documentation.

I'll add a demo for this :+1:

Reply:
It would seem that sphinx-automodapi generates the same outputs as autosummary, so I will skip it entirely and just use autosummary :+1: 

Thanks for being quick!